pom.xml

```diff
@@ -65,12 +65,25 @@
             <artifactId>branch-api</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>github-pr-label-build</artifactId>
+            <version>2.0-custom-plugin</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>jackson2-api</artifactId>
+            <version>2.14.1-313.v504cdd45c18b</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>credentials</artifactId>
+            <version>1214.v1de940103927</version>
+        </dependency>
     </dependencies>
 
     <properties>
-        <revision>1.10</revision>
-        <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.361.4</jenkins.version>
+        <jenkins.version>2.375</jenkins.version>
```

Comment:
That's the version on `green` will this be compatible?
```
 2.387.1
```

Reply:
Jest it is.

Reply:
Yes, it is. I have upgraded it a little so it conforms with a gh pr label plugin for which the latest dependencies forced to update the jenkins lib to at least version 2.375